doc/tutorial.md

```diff
@@ -1,3 +1,259 @@
-# IAMF tutorial
+###Constructing A One-Region Model
+---
 
-TODO
+In this example, we will construct a stylized model of the global economy and its changing greenhouse gas emission levels through time. The overall strategy will involve creating components for the economy and emissions separately, and then defining a model where the two components are coupled together.  
+
+There are two main steps to creating a component:
+- [ ] Define the component using ``@defcomp`` where the parameters and variables are listed.
+- [ ] Use the timestep function ``timestep(state::component_name, t:Int)`` to set the equations of that component.  
```

Comment:
Ah, interesting... This might be a quirk of mkdocs. But I think (not sure though) that the final rendering will be done according to mkdocs rules, so we should probably make sure it looks good with that.